### PR TITLE
test:TD-22428

### PR DESCRIPTION
**Pull request by cadem**

### CODE REVIEW

File: tests/system-test/2-query/schema_limit.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        tdSql.query(sqlStr)
+        tdSql.checkRows(5)
+        sqlStr = f"select * from information_schema.ins_tables where db_name='%s' limit 10 offset 0;"%(paraDict["dbName"])
+        print("====sql:%s"%(sqlStr))
+        tdSql.query(sqlStr)
+        tdSql.checkRows(10)
+        sqlStr = f"select * from information_schema.ins_tables where db_name='%s' limit 15 offset 0;"%(paraDict["dbName"])
+        print("====sql:%s"%(sqlStr))
+        tdSql.query(sqlStr)
+        tdSql.checkRows(15)
```
Comment (jiacy-jcy) [inline tests/system-test/2-query/schema_limit.py:126]: 111-122行，可以将limit的值作为传参，这样这部分代码只需要描述一次，可以显得更精简。

**Review by happyguoxy — COMMENTED** [review]
,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -R
,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 2
,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 3
,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 4 
是否能够执行

File: tests/system-test/2-query/schema_limit.py (hunk 1 above)
Comment (cadem) [reply tests/system-test/2-query/schema_limit.py:126]: refactor done

**Comment by cadem** [comment]
> ,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -R ,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 2 ,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 3 ,,y,system-test,./pytest.sh python3 ./test.py -f 2-query/schema_limit.py -Q 4 是否能够执行

Test pass with R&Q

**Pull request closed without merging**